ee/clickhouse/models/test/test_cohort.py

```diff
@@ -339,6 +340,28 @@ def test_cohortpeople_basic(self):
         )
         self.assertEqual(len(results), 2)
 
+    @patch("posthog.tasks.calculate_cohort.calculate_cohort")
+    def test_cohortpeople_basic_paginating(self, mock_calculate_people):
+        for i in range(50):
```

Comment:
Any specific reason for 50? Is it the minimum reproducible case for the feature you are testing for? I would generally try to avoid arbitrary numbers, but be very deliberate about them to try and completely focus on what you're testing.

if there's something specific about "I want to make sure that paginating 10 times on pg works and doesn't hit this specific case" then maybe it's worth having a specific test for that?

Would `batch_size=1, pg_batch_size=1` and `range(2)` also test the pagination sufficiently? Or if we think that there is a risk that we might write a bug that would only get the first item per pagination, then `batch_size=2, pg_batch_size=2` and `range(4)` ?  Or perhaps we want to make sure that we handle partial pages of results as well.